doc/spinel-protocol-src/spinel-tech-thread.md

```diff
@@ -233,3 +233,24 @@ The leader network data.
 * Packed-Encoding: `D`
 
 The stable leader network data.
+
+### PROP 5391: PROP_THREAD_JOINERS {#prop-thread-joiners}
+
+* Type: Read-Write
+* Packed-Encoding: `A(T(EDc))`
+
+Data per item is:
+
+* `E`: Extended/long address
+* `D`: PSKd
+* `c`: PSKd length
```

Comment:
`c` is unnecessary. `D` includes a length.

But your confusion on this is entirely understandable. I'll be changing this around soon to simplify the behavior of `D`. Will elaborate more in another comment.